spec/controllers/pages_controller_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe HighVoltage::PagesController do
```

Comment:
I'm OK with it if you want to disable rubocop on this one.  I can't find the exact rule name, though.